src/main/java/train/common/core/handlers/ItemHandler.java

```diff
@@ -45,6 +42,11 @@ public static boolean handleItems(Entity entity, ItemStack itemstack) {
 	}
 
 	public static boolean handleFreight(Entity entity, ItemStack itemstack) {
+        int logWood = OreDictionary.getOreID("logWood");
```

Comment:
You might wanna double check TC code yourself @duchuyctlk , theres a significant number of inconsistencies in the code already, from different whitespacing to bracket useage. This project has no real direction for code organization.